handbook/sales.md

```diff
@@ -10,7 +54,7 @@ Directly Responsible Individuals (DRI) engage in the ritual(s) below at the freq
 | Ritual                       | Frequency                | Description                                         | DRI               |
 |:-----------------------------|:-----------------------------|:----------------------------------------------------|-------------------|
 | Sales huddle | Weekly | Agenda: Go through every [open opportunity](https://fleetdm.lightning.force.com/lightning/o/Opportunity/list?filterName=00B4x00000CTHZIEA5) and update next steps. | Alex Mitchell
-| TODO  | TODO | TODO | TODO 
+[Salesforce contributor experience checkup](./growth.md#salesforcecontributorexperiencecheckups)| Monthly | Make sure all users see a detailed view of contacts, opportunities, accounts, and leads. | Nathan Holliday |
```

Comment:
Should this be pointed to the sales handbook page instead of the growth page since the content has moved?

Reply:
Tracked here: https://github.com/fleetdm/fleet/pull/6105 and here https://github.com/fleetdm/fleet/pull/6104 (it was merged before I got your confirmation to fully remove from Growth to Sales. 

These PRs are merged already.